### DISCUSSION TITLE
HP ProBook 4740S

### DISCUSSION THREAD

**Discussion by DKD85** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#95] [Q&A]
Hi, chris1111.
I want to install Mac OS BigSur or Monterey using your guid. 
Will I be able to update Mac OS easily throughout Software Update after installation?

**Comment by chris1111** [comment]
Yes

**Reply by DKD85** [reply]
Good news!!!! Thanks for your answer!!!!
I have two last questions: 
First: Will a speedstep and turboboost work for my 3210M on my laptop?
Second: Will a card-reader work on Monterey?

**Comment by chris1111** [comment]
Card reader works same like Big Sur, see How to
Speed steps [Read this](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator-OC/issues/77)

**Reply by DKD85** [reply]
Thanks a lot!

**Comment by DKD85** [comment]
Hi, chris1111.
I've come up with one more question. Should SIP be permanently activated after installation?
Thanks!

**Reply by chris1111** [reply]
> Hi, chris1111. I've come up with one more question. Should SIP be permanently activated after installation? Thanks!

Yes for Big Sur but not for Monterey because your on SnapShot disk for HD 4000 patch
There is no way booting Snapshot disk without csrutil authenticated-root disable
(csr-active-config ➤ EF0F0000

**Reply by DKD85** [reply]
Thanks for the answer. So, if I  understand you correctly, I can Install MacOS Monterey, then tern off SIP. After that install HD 4000 patch and then tern SIP on. Am I right?

**Comment by chris1111** [comment]
No the SIP must always be deactivated otherwise you will not be able to boot from a Shapshot disk with kext patched

**Reply by DKD85** [reply]
Ok, thanks, understood!
